docs/manage/_get_support.rst~HEAD

```diff
@@ -0,0 +1,62 @@
+.. _support:
```

Comment:
this file (`~HEAD`) shouldn't be here

Reply:
Says a lot about the number of conflicts :)